docs/api/config/js_kanban_cardshape_config.md

```diff
@@ -20,6 +20,7 @@ cardShape?: {
     attached?: boolean | { show?: boolean },
     cover?: boolean | { show?: boolean },
     comments?: boolean | { show?: boolean },
+    confirmDeletion?: boolean,
```

Comment:
cardShape was correct in the very 1st commit. in cardShape, confirmDeletion is boolean | { show: boolean }
it's other shapes that have it as boolean.